.github/workflows/ci-integration-review.yml

```diff
@@ -112,109 +105,109 @@ jobs:
           command: bash scripts/ci-int-e2e-test.sh int amqp
```

Comment:
Besides the core of this PR, but related to the original reason ("vulnerable GH workflows"): given the architecture of these invocations, you might want add `scripts/` and the `.github/CODEOWNERS` themselves to CODEOWNERS: https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-and-branch-protection

Obviously depends on how externals can become contributors and whether there is review on every change from external.

Reply:
I'm not sure I understand this one. Someone on `@vectordotdev/vector` reviews all external changes regardless. It is reasonable enough to add ownership for all files though. I'll open a separate PR for that.

Reply:
Opened https://github.com/vectordotdev/vector/pull/20947